docs/config-file/node-config-doc.md

```diff
@@ -1215,6 +1216,118 @@ requests to be captured by the server.
 EnableHttpLog=true
 ```
 
+### <a name="RPC_ZKCountersLimits"></a>8.17. `[RPC.ZKCountersLimits]`
```

Comment:
ZKCountersLimits shouldn't be documented as config parameter for the RPC section, since really it's not a config section,  it uses the values from State.Batch.Constraints